data/card_catalogue/unique-1.json

```diff
@@ -64,7 +44,7 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Target a large object or willing NPC and return this card to the deck after placing them in your inventory.",
+      "rules": "Place target large object or willing NPC in your hand.",
```

Comment:
We don't really have an 'inventory', and we don't need this to be single use. This wording is a little more concise